tests/test_data_generators.py

```diff
@@ -0,0 +1,122 @@
+import os
```

Comment:
Cool! I like these tests 👌  They are simple, but they touch a lot of the code which is what you care about in this stage.

Reply:
Nice, thanks.